Gruntfile.js

```diff
@@ -3,35 +3,62 @@
  * https://github.com/wildbit/grunt-postmark.git
  */
 
-module.exports = function(grunt) {
+module.exports = function (grunt) {
+  var secret = grunt.file.readJSON('secrets.json');
+  var config = grunt.file.readJSON('config.json');
 
   grunt.initConfig({
-    secrets: grunt.file.readJSON('secrets.json'),
+    secret: secret,
+    config: config,
 
     /* Postmark
-    ------------------------------------------------- */
+     ------------------------------------------------- */
     postmark: {
       options: {
-        serverToken: '<%= secrets.serverToken %>'
+        serverToken: "<%= secret.postmark.server_token %>",
       },
       email: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
-        subject: 'Yo',
+        from: "<%= config.postmark.from %>",
+        to: "<%= config.postmark.to %>",
+        subject: "<%= config.postmark.subject %>",
         src: ['test/email.html']
       },
       bulk: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
-        subject: 'Hey',
+        from: "<%= config.postmark.from %>",
+        to: "<%= config.postmark.to %>",
+        subject: "<%= config.postmark.subject %>",
         src: ['test/*.html']
       }
-    }
+    },
+
+    // you can either specify the template configuration here, or in templates.json
+    'postmark-templates-upload': config.templates && config.templates
+      ? config.templates.output_file || config.templates.file
+      : {
+        test_email: {
+          name: 'testing-postmark-templates-js1-' + new Date().valueOf(),
+          subject: 'Testing grunt-postmark-templates',
+          htmlBody: 'test/email.html',
```

Comment:
This looks like it's simply writing the string to the Postmark template instead of reading and posting the contents of the file.

Am I missing something about the setup here?

<img width="856" alt="screen shot 2017-11-24 at 10 24 21 am" src="https://user-images.githubusercontent.com/3054417/33220004-b2eed864-d102-11e7-8b7f-26e352d0f33d.png">